apps/amo/management/commands/clean_redis.py

```diff
@@ -1,14 +1,12 @@
 import logging
-import os
 import socket
-import subprocess
-import sys
-import tempfile
 import time
 
+from django.conf import settings
+from django.core.cache import parse_backend_uri
 from django.core.management.base import BaseCommand
 
-import redisutils
+# import redisutils
```

Comment:
I am waiting for https://github.com/washort/nuggets/pull/1 to be merged in so that I can reuse redisutils instead of having my own get_redis_backend()